docker-compose.dev.yml

```diff
@@ -30,6 +30,22 @@ services:
   resource-recorder:
     ports:
       - 8007:8000
+    depends_on:
+      - auth
```

Comment:
Nit: The .dev.yml file will be combined with the docker-compose.yml into docker-compose.rendered.yml for local development, so we don't need to specify the auth dependency here. I see we did the same thing for logger in this file.

Reply:
9efbed5845cb18cdfd3071a203abf99a5549ceda